common/views/components/SearchForm/SearchForm.tsx

```diff
@@ -1,168 +1,87 @@
-import {
-  useRef,
-  useState,
-  useEffect,
-  ReactElement,
-  forwardRef,
-  useImperativeHandle,
-} from 'react';
-import Router from 'next/router';
-import styled from 'styled-components';
-import { ParsedUrlQuery } from 'querystring';
-
-import TextInput from '@weco/common/views/components/TextInput/TextInput';
-import ButtonSolid from '@weco/common/views/components/ButtonSolid/ButtonSolid';
-import { trackGaEvent } from '@weco/common/utils/ga';
-import Space from '@weco/common/views/components/styled/Space';
-import ClearSearch from '@weco/common/views/components/ClearSearch/ClearSearch';
-import { searchFormInputCatalogue } from '@weco/common/text/aria-labels';
-import { LinkProps } from '@weco/common/model/link-props';
+import { useRouter } from 'next/router';
+import SearchBar, {
+  ValidLocations,
+} from '@weco/common/views/components/SearchBar/SearchBar';
+import { linkResolver, getQueryPropertyValue } from '@weco/common/utils/search';
 import { formDataAsUrlQuery } from '@weco/common/utils/forms';
-
-const Wrapper = styled(Space).attrs({
-  h: { size: 'm', properties: ['padding-left', 'padding-right'] },
-  v: { size: 'm', properties: ['padding-top', 'padding-bottom'] },
-})`
-  display: flex;
-`;
-
-const SearchInputWrapper = styled.div`
-  position: relative;
-  flex: 1 1 auto;
-
-  .search-query {
-    height: ${props => 10 * props.theme.spacingUnit}px;
+import { useState, useRef, useEffect, useContext, ReactElement } from 'react';
+import SearchContext from '@weco/common/views/components/SearchContext/SearchContext';
+
+type SearchCategory = 'all' | 'works';
+
+function placeholderText(searchCategory: SearchCategory) {
+  switch (searchCategory) {
+    case 'all':
+      return 'Search our stories, images and catalogue';
+    case 'works':
+      return 'Search our catalogue and images';
```

Comment:
As it goes on `/works`, should it just say "Search our catalogue"?

Reply:
I had the same thought. This was in the designs, but worth checking. @DominiqueMarshall what do you think?

Reply:
Yep - it should say "Search the catalogue"